### ISSUE TITLE
[DCS] unable to create single instance

### ISSUE THREAD

**Issue by sfhl** [opentelekomcloud/terraform-provider-opentelekomcloud#639]
Hi there,

there is a problem creating DCS (redis) in single node mode. The provider wants backup parameters to be defined, but the api gives an error if so (see also https://docs.otc.t-systems.com/en-us/api/dcs/dcs-api-0312004.html), so you are nto able to create a single node DCS. Master/Standby & Cluster is working fine.

# Terraform Version
```
Terraform v0.12.24
+ provider.opentelekomcloud v1.20.2
```

# Affected Resource(s)
- opentelekomcloud_dcs_instance_v1

# Terraform Configuration Files
```hcl
resource "opentelekomcloud_dcs_instance_v1" "dcs-prod-project-app-backend-test" {
  name        = "dcs-prod-project-app-backend-test"
  description = ""

  engine         = "Redis"
  engine_version = "3.0.7"
  password       = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
  capacity       = "2"

  security_group_id = opentelekomcloud_networking_secgroup_v2.sg-prod-default.id
  subnet_id         = opentelekomcloud_vpc_subnet_v1.prod-project-app.id
  vpc_id            = opentelekomcloud_vpc_v1.prod-project.id

  product_id      = data.opentelekomcloud_dcs_product_v1.dcs_product_dcs-prod-project-app-backend-test.id
  available_zones = [data.opentelekomcloud_dcs_az_v1.dcs_az_dcs-prod-project-app-backend-test_eu-de-01.id]

  backup_type = "manual"
  period_type = "weekly"
  save_days   = "1"
  begin_at    = "00:00-01:00"
  backup_at   = ["1"]

  depends_on = [opentelekomcloud_networking_secgroup_v2.sg-prod-default]
}
```

# Debug Output/Panic Output
```
...
Error: Error creating instance: Bad request with: [POST https://dcs.eu-de.otc.t-systems.com/v1.0/2e0c0227665749b99c45df08f931611b/instances], error message: {"error":{"code":"111400086","message":"only master standby instance is support this action","ext_message":null}}

  on main.tf line 302, in resource "opentelekomcloud_dcs_instance_v1" "dcs-prod-project-app-backend-test":
 302: resource "opentelekomcloud_dcs_instance_v1" "dcs-prod-project-app-backend-test" {
...
```

# Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

# Expected Behavior
Create DCS

# Actual Behavior
Error shown

---

If manually created and imported, the following changes are targeted:

```
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
-/+ destroy and then create replacement

Terraform will perform the following actions:

  # opentelekomcloud_dcs_instance_v1.dcs-prod-project-app-backend-test must be replaced
-/+ resource "opentelekomcloud_dcs_instance_v1" "dcs-prod-project-app-backend-test" {
      + available_zones     = [
          + "bf84aba586ce4e948da0b97d9a7d62fb",
        ] # forces replacement
      + backup_at           = [
          + 1,
        ] # forces replacement
      + backup_type         = "manual" # forces replacement
      + begin_at            = "00:00-01:00" # forces replacement
        capacity            = 2
      + description         = (known after apply)
        engine              = "Redis"
        engine_version      = "3.0"
      ~ id                  = "46b793b9-174a-4681-a4fa-e5b690bf4b32" -> (known after apply)
      + internal_version    = (known after apply)
      + ip                  = (known after apply)
      ~ maintain_begin      = "02:00:00" -> (known after apply)
      ~ maintain_end        = "06:00:00" -> (known after apply)
      + max_memory          = (known after apply)
        name                = "dcs-prod-project-app-backend-test"
      + order_id            = (known after apply)
      + password            = (sensitive value)
      + period_type         = "weekly" # forces replacement
      ~ port                = 6379 -> (known after apply)
        product_id          = "OTC_DCS_SINGLE"
      ~ resource_spec_code  = "dcs.single_node" -> (known after apply)
      + save_days           = 1 # forces replacement
      ~ security_group_id   = "6ea5deac-724c-45b5-9048-15177392de6c" -> "bf4b77c9-31ac-4e95-a43b-197713aa720b"
      ~ security_group_name = "default" -> (known after apply)
        subnet_id           = "d7b00843-6e19-4ca9-a0da-0b4350ca3224"
      ~ subnet_name         = "app" -> (known after apply)
      ~ used_memory         = 0 -> (known after apply)
      ~ user_id             = "TksMKO1hCS4MQMw1Hu5kHNmgHmTUXp3X" -> (known after apply)
        vpc_id              = "1536ed34-c2a6-4a98-8100-3c24346a6dbd"
      ~ vpc_name            = "prod-project" -> (known after apply)
    }
```

**Comment by outcatcher** [comment]
Fix will be part of the next release

**Comment by outcatcher** [comment]
Implemented in `1.20.3`.
@sfhl please check if it works as expected.

**Comment by sfhl** [comment]
Hi there,
seems to work as expected, thank you very much!
